public/js/components/Scopes.js

```diff
@@ -120,10 +120,18 @@ function getScopes(pauseInfo, selectedFrame) {
         scopes.push({ name: title, path: key, contents: vars });
       }
     } else if (type === "object") {
+      const value = scope.object;
+      // Override preview for Window object
+      if (value.class === "Window") {
+        value.preview = {
+          kind: "Window",
+          isGlobal: true
+        };
+      }
```

Comment:
Yeah, I think we should avoid it. We want to be careful about mutations because they cause surprising effects down the road. It might be good to ask Honza about this.